storage-proofs-update/src/vanilla.rs

```diff
@@ -901,30 +907,40 @@ where
                     let high = node_index >> get_high_bits_shr;
                     let rho = rhos[high];
 
-                    let sector_key_fr =
```

Comment:
Note: `fr32`'s bytes to/from field conversion functions are only defined for `Fr`